format-translators-module/format-translators-module/model/link.cpp

```diff
@@ -0,0 +1,36 @@
+#include "link.hpp"
+
+#include "visitor/visitor.hpp"
+
+namespace formatTranslators
+{
+void Link::AcceptVisitor(Visitor & visitor)
```

Comment:
Please, sort functions implementations according used order of their definitions in hpp file